api/api/src/test/scala/uk/ac/wellcome/platform/api/services/ImagesServiceTest.scala

```diff
@@ -47,7 +47,7 @@ class ImagesServiceTest
   }
 
   describe("retrieveSimilarImages") {
-    it("gets visually similarImages") {
+    ignore("gets visually similarImages") {
```

Comment:
Is this test coming back later? Maybe either remove it or add a comment why it's ignored.

Reply:
I'm working on a PR that includes this right now

Reply:
Just left a note, @jamieparkinson is on writing more meaningful features for this, so waiting on that.

It fails given the version we are running in prod. Makes me think running `latest` elastic containers could be a good thing... Or be more conservative when updating the catalogue cluster.

Reply:
:+1: